spec.html

```diff
@@ -4585,6 +4585,40 @@ <h1>The ClassFieldDefinition Record Specification Type</h1>
       <h1>Private Names</h1>
       <p>The <dfn>Private Name</dfn> specification type is used to describe a globally unique value (one which differs from any other Private Name, even if they are otherwise indistinguishable) which represents the key of a private class element (field, method, or accessor). Each Private Name has an associated immutable [[Description]] which is a String value. A Private Name may be installed on any ECMAScript object with PrivateFieldAdd or PrivateMethodOrAccessorAdd, and then read or written using PrivateGet and PrivateSet.</p>
     </emu-clause>
+
+    <emu-clause id="sec-classstaticblockdefinition-record-specification-type">
+      <h1>The ClassStaticBlockDefinition Record Specification Type</h1>
+      <p>A <dfn>ClassStaticBlockDefinition Record</dfn> is a Record value used to encapsulate the executable code for a class static initialization block.</p>
```

Comment:
```suggestion
      <p>A <dfn variants="ClassStaticBlockDefinition Record">ClassStaticBlockDefinition Record</dfn> is a Record value used to encapsulate the executable code for a class static initialization block.</p>
```

(We [just now](https://github.com/tc39/ecma262/pull/2492) added support for auto-linking plural forms when specified like this.)